### DISCUSSION TITLE
If I am admiring the "topiary" what am I looking at?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#878] [Galaxy Trivia] [answered]
*answer with a picture*

**Comment by peterbaccaro** [comment, marked as answer]
If you are admiring a "topiary," you are looking at a form of ornamental gardening or artistry that involves shaping and trimming living plants, typically trees or shrubs, into intricate and decorative designs. Topiary is the practice of sculpting plants into various shapes, such as geometric forms, animals, or even human figures. This art form requires careful pruning and training of the plants to maintain the desired shape. Topiary can be found in gardens, parks, and sometimes even indoors as decorative elements.

![image](https://github.com/ParanoidUser/yolo/assets/74666079/32b99895-1ab7-4346-9a45-0ba341633d47)
